ROOTFS/etc/systemd/system/waggle-wan-tunnel.service

```diff
@@ -2,6 +2,9 @@
 Description=Waggle WAN Tunnel
 
 [Service]
+Type=notify
+NotifyAccess=all
+TimeoutStartSec=30s
```

Comment:
This is to protect against the corner case where sshuttle doesn't start correctly.